### PR TITLE
Setup DynamoDB

### PR DESCRIPTION
**Pull request by aomi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,4 +5,40 @@ Documentation is work in progress!
...
 2. Run `main.py` using `pipenv run python main.py`
-    - **Note**: The bot expects either the `BOT_TOKEN` passed in as an environement variable or within a `.env` file in the project root directory. 
\ No newline at end of file
+    - **Note**: The bot expects either the `BOT_TOKEN` passed in as an environement variable or within a `.env` file in the project root directory. 
+
+### Database
+VikeBot uses DynamoDB, which is a NoSQL database by Amazon. Below provides some brief instructions on how to setup a local instance of DynamoDB as well as a visual tool to interact with the database.
+
+1. Install a local DynamoDB instance.
+    - This can be anything but for the same of simplificty, we'll be using the [AWS-provided Java version](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html).
```
Comment (k-kali) [inline README.md:14]: Small typo here

File: README.md [hunk 2]
```diff
@@ -5,4 +5,40 @@ Documentation is work in progress!
...
+    - For those who have Docker installed, that will probably be easier.
+    - Unzip the contents of the `zip` or `tar.gz` file into a folder called `tmp` in the repository root folder.
+    - `unzip dynamodb_local_latest.zip -d tmp`
+2. Start the local DynamoDB instance.
+    - `java -Djava.library.path=./tmp/DynamoDBLocal_lib -jar ./tmp/DynamoDBLocal.jar -sharedDb` from the repository directory.
+    - This will start the instance on port `8000`. Please read the AWS documentation for changing the port.
+    - Congrats, you have a local DynamoDB instance!
+
+**Optional**: If you wish to have a visual interface to interfact with DynamoDB, follow these steps!
```
Comment (k-kali) [inline README.md:23]: Another small typo "interfact"

File: README.md [hunk 3]
```diff
@@ -5,4 +5,40 @@ Documentation is work in progress!
...
+```
+# For Windows:
+set DYNAMO_ENDPOINT=http://localhost:8000
+npx dynamodb-admin
+
+# For Mac/Linux:
+DYNAMO_ENDPOINT=http://localhost:8000 npx dynamodb-admin
+```
+
+Once the all of the above is setup, try playing around using the Python interrupter.
```
Comment (k-kali) [inline README.md:35]: Another small type "interrupter"

**Review by k-kali — COMMENTED** [review]
I noted some typos throughout the readme file. Other than that things look good to me, but Sam might be a better person to approve this.

**Review by SwammyBoy — APPROVED** [review]
Content discussed in extra meeting - this looks good to me

**Pull request merged**